Description: add fix for terser >= 5
Author: Yadd <yadd@debian.org>
Bug-Debian: https://bugs.debian.org/1018834
Forwarded: no
Last-Update: 2022-08-31

--- a/bundt/index.js
+++ b/bundt/index.js
@@ -31,7 +31,7 @@
 	return out + ' ' + UNITS[exp];
 }
 
-function write(file, data, isUMD, toDir) {
+async function write(file, data, isUMD, toDir) {
 	file = normalize(file);
 	let isDef = /\.d\.ts$/.test(file);
 	if (isDef && toDir !== 'default') {
@@ -41,7 +41,7 @@
 	}
 	mkdir(dirname(file)); // sync
 	let code = isDef && data;
-	code = code || minify(data, Object.assign({ toplevel:!isUMD }, terser)).code;
+	code = code || (await minify(data, Object.assign({ toplevel:!isUMD }, terser))).code;
 	writeFileSync(file, isUMD ? code : data);
 	let gzip = size(gzipSync(code).length);
 	return { file, size: size(code.length), gzip };
